docs-src/modules/ROOT/pages/core/parsers.adoc

```diff
@@ -1,8 +1,180 @@
 = Parsers
 
+Parsers are the core of what Pathom does. This section explains what Pathom parser does,
```

Comment:
Parsers are at the core of what Pathom does. This section explains how Pathom parsers accomplish this responsibility and describes how they work.